### PR TITLE
Improves README for self-surface.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes https://github.com/entropyxyz/cli/issues/19

This PR doesn't contain 

- troubleshooting steps yet, because we don't have any.
- contribution instructions, because the JS team are figuring out how to manage that. Don't wanna put words in their mouth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,113 @@
...
+### Stop the CLI
+
+You can stop the CLI by selecting **Exit** from the main menu or pressing `CTRL` + `c`.
+
+### Available functions
+
+| Function | Description |
+| -------- | ----------- |
+| Manage accounts | Create, import, and list your locally stored Entropy accounts. |
+| Balance | Show the balance of any locally stored accounts. |
```
Comment (rh0delta) [inline README.md:98]: Balance shows only the free balance of the currently selected account, might want to change the wording on this one

File: README.md [hunk 2]
```diff
@@ -1,71 +1,113 @@
...
+| Register | Register a locally stored account with the Entropy network. |
+| Sign | Get a signature from the Entropy network using a locally stored, registered account. |
+| Transfer | Transfer funds from a locally stored account to any other address. |
+| Deploy Program | Deploy a program to the Entropy network using a locally stored account. |
+| User Programs | List the programs stored on the Entropy network by any locally stored accounts. |
```
Comment (rh0delta) [inline README.md:103]: same here the list of programs is for the currently selected account

**Review by rh0delta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (johnnymatthews) [reply README.md:98]: Yeah you're right. But I don't want to complicated this readme by getting into the whole _selecting an account_ thing. That's discussed over in the docs. For now, I reckon this content is fine.

File: README.md (hunk 2 above)
Comment (johnnymatthews) [reply README.md:103]: Duplicated from [previous comment](https://github.com/entropyxyz/cli/pull/100#discussion_r1637064425):

> Yeah you're right. But I don't want to complicated this readme by getting into the whole _selecting an account_ thing. That's discussed over in the docs. For now, I reckon this content is fine.

**Review by rh0delta — APPROVED** [review]

**Pull request merged**